### ISSUE TITLE
Invalid Field Id: customfield_10985

### ISSUE THREAD

**Issue by packetfairy** [tenable/integration-jira-cloud#274]
I realize this is [another duplicate of an issue that many people seem to have had](https://github.com/tenable/integration-jira-cloud/issues/23). But evidently I'm slow.

Can someone please help me understand which `config.yaml` file we're all editing to resolve the issue with accessing jira screens?

I accept that I'm thick, but the config template in the project root is a TOML file, not YAML. So.. are we converting the entire file to YAML by hand first?

Alternatively, can someone please help me understand how I would make this same configuration change using the TOML file?

**Comment by Flv-cmd** [comment]
> I realize this is [another duplicate of an issue that many people seem to have had](https://github.com/tenable/integration-jira-cloud/issues/23). But evidently I'm slow.
> 
> Can someone please help me understand which `config.yaml` file we're all editing to resolve the issue with accessing jira screens?
> 
> I accept that I'm thick, but the config template in the project root is a TOML file, not YAML. So.. are we converting the entire file to YAML by hand first?
> 
> Alternatively, can someone please help me understand how I would make this same configuration change using the TOML file?

If I understand correctly, this ID field is not present in your jira, but it is written in config.toml. You need to either map the ID in jira itself using the name field, or clear the IDs in config.toml and then run the config and they will be registered automatically. 
In jira go to settings>Issues>Custom Fields and check  ID number. 
If this ID does not exist, create it yourself. Or open config.toml and at the very bottom in
 [[jira.fields]]  erase 
id = "customfield_100XXX"

**Comment by SteveMcGrath** [comment]
The TOML file is whats used for the v2 Jira integration.  The integration will attempt to map any existing fields based on name to the fields it needs, and then will create them if necessary.  If you could provide the log entry of the error in question, that would help.

**Comment by packetfairy** [comment]
Ok, freekin weirdweird.

I was using a config file that was handed off to me by another engineer who was struggling to make things work. After receiving different errors (I don't know what they were), he kinda threw his hands up and had to move on to other things. So the file was already modified with the customfield IDs when I got it.

After noting the suggestion above that I erase those lines, I (finally) compared the config file I received with the template in the project root, and discovered the unexpected (at least to me) differences.

Ever more confused (and certain that I would still encounter errors), I decided to start from scratch: I populated a new template file with the necessary config values, ran build, and it worked perfectly. Well, son of a bitch. I bet sync fails! Nope, that works too. Well, double son of a bitch.

So, I don't know why it was failing for my teammate. And I also don't know why it was failing for me. But it's no longer failing. So.. yay for magic!

Thanks for the quick replies. They clearly got me where I needed to be.
